src/components/atoms/SearchBar/index.jsx

```diff
@@ -83,11 +80,12 @@ class SearchBar extends React.Component {
       >
         <MDBSelectInput selected="Find a user" />
         <MDBSelectOptions search searchLabel="">
-          {this.state.usernames ? (
+          {!this.state.loading && this.state.usernames ? (
             this.state.usernames.length > 0 && this.state.filter.length > 0 ? (
               fuzzysort
                 .go(this.state.filter, this.state.usernames)
                 .map((element, i) => {
+                  console.log("FUZZY ELEMNET", element);
```

Comment:
Please remove or tag as DEBUG.

```suggestion
                  console.log("FUZZY ELEMNET", element);
```